server/src/test/java/io/crate/integrationtests/DynamicMappingUpdateITest.java

```diff
@@ -191,6 +191,23 @@ insert into t values (1, {x=1})
             "b['newcol84']",
             "b['x']"
         );
+
+        // Verify that there is no holes in positions sequence for a concrete table
+        execute("""
+            SELECT
+                column_name, ordinal_position
+            FROM
+                information_schema.columns
+            WHERE
+                table_name = 't'
+            AND
+                ordinal_position NOT IN (SELECT * FROM generate_series(1, 48))
```

Comment:
Minor: How about `> 48`?
Also, maybe making this into a concurrent task which could identify re-computed positions could be valuable.

Reply:
> How about > 48?

Yep, it's much simpler, thanks!

> Also, maybe making this into a concurrent task which could identify re-computed positions could be valuable.

Could you please elaborate? Do you mean adding such check to each `concurrentUpdates` thread?